routes/tasks.ts

```diff
@@ -1,21 +1,25 @@
-import connectDB from "../db/conn";
-import {Collection, Db, WithId} from "mongodb";
 import express, {Request, Response, NextFunction} from "express";
+import mongoose, {Schema} from "mongoose";
+
+mongoose.connect(process.env.ATLAS_URI as string)
+    .then(() => console.log('Connected to MongoDB'))
+    .catch((err) => console.error('Failed to connect to MongoDB', err));
+
+const taskSchema: Schema = new Schema({
```

Comment:
Would be nice to separate schemas from task related logic, you could create a models folder for the schemas